src/DotLiquid/StandardFilters.cs

```diff
@@ -976,6 +999,46 @@ private static bool HasMatchingProperty(this object any, string propertyName, ob
                 ? propertyValue.IsTruthy()
                 : propertyValue.SafeTypeInsensitiveEqual(targetValue);
         }
+
+        /// <summary>
+        /// Concatenates (joins together) multiple arrays.
+        /// The resulting array contains all the items from the input arrays.
+        ///
+        /// Will not remove duplicate entries from the concatenated array
+        /// unless you also use the uniq filter.
+        /// </summary>
+        /// <param name="left">left hand (start) of the new concatenated array</param>
+        /// <param name="right">array to be appended to left</param>
+        /// <returns></returns>
+        /// <see href="https://shopify.github.io/liquid/filters/concat/"/>
+        public static IEnumerable Concat(IEnumerable left, IEnumerable right)
+        {
+            // If either side is null, return the other side.
+            if (left == null)
+                return right;
+            else if (right == null)
+                return left;
+
+            List<object> leftList = left.Cast<object>().ToList();

```

Comment:
likely you can use `var` here and on next line. And as the variables are used only once, you should consider just inlining the whole thing.

Reply:
Inlined.